src/cookbooks/bigquery-airflow.md

```diff
@@ -0,0 +1,76 @@
+# Scheduling BigQuery Queries in Airflow
+
+Queries in [BigQuery-ETL](https://github.com/mozilla/bigquery-etl) can be scheduled in 
```

Comment:
I'd prefer to consistently style this as `bigquery-etl` rather than `BigQuery-ETL` to match usage elsewhere.